src/mapgen/mg_decoration.cpp

```diff
@@ -87,6 +87,9 @@ void Decoration::resolveNodeNames()
 
 bool Decoration::canPlaceDecoration(MMVManip *vm, v3s16 p)
 {
+	// This functions coordinate 0,0,0 is the node the decoration will be placed ontop of
+	// Not the decoration itself!
```

Comment:
I'm a bit confused about githubs UI here, it sais to commit suggestion... but it wants to create a new commit instead of updating the to-be-commited one? Seems a bit silly to commit all wrong iterations of a change?

Reply:
The suggested line is too long according to the guidelines